hooks/_common.sh

```diff
@@ -206,8 +206,10 @@ function common::get_cpu_num {
 
   local cpu_quota cpu_period cpu_num
 
+  local wslinterop_path="/proc/sys/fs/binfmt_misc/WSLInterop"
```

Comment:
Let's make it read-only 


```suggestion
  local -r wslinterop_path="/proc/sys/fs/binfmt_misc/WSLInterop"
```

Reply:
Makes sense. Thanks.